src/baseballGame.js

```diff
@@ -0,0 +1,72 @@
+const { RESULT, GAME_MESSAGE, GAME_ERROR_MESSAGE } = require("./constants.js");
+const {generateRandomNumber, print} = require("./util.js");
+const { Console } = require("@woowacourse/mission-utils");
+const checkUserInput = require("./userInputCheck.js");
+
+class BaseballGame {
+  constructor() {
+    this.computerNumber = this.computerNumber;
+    this.userInput = this.userInput;
+  }
+
+  play() {
+    print(GAME_MESSAGE.START);
+    this.computerNumber = generateRandomNumber();
+    this.readUserInput(GAME_MESSAGE.INPUT);
+  }
+
+  readUserInput(message) {
+    Console.readLine(message, (userInput) => {
+      if (message === GAME_MESSAGE.RESTART)
+        return this.choiceReplayOrClose(userInput);
+      this.userInputHandling(userInput);
+      this.readUserInput(GAME_MESSAGE.INPUT);
+    });
+  }
```

Comment:
콜백을 외부로 분리해주시면 더 좋을 것 같아요! 
if문을 두줄로 쓸때는 브라켓을 넣어주시는게 좋을 것 같아요!:) 혹은 아래처럼은 어떤가요?:) 
```suggestion
    Console.readLine(message, (userInput) => {
      if (message === GAME_MESSAGE.RESTART)  return this.choiceReplayOrClose(userInput);

      this.userInputHandling(userInput);
      this.readUserInput(GAME_MESSAGE.INPUT);
    });
  }
```

Reply:
return 문을 한줄에 작성하는 것도 좋네요 !! 콜백을 외부로 분리하는것도 이번 주차에서는 시도해보겠습니다 ✊🏻